src/system.h

```diff
@@ -906,6 +906,12 @@ typedef struct {
     const Char *    cookie;
 } StructGVarFunc;
 
+#define GAP_GVAR_FUNC_TABLE_ENTRY(srcfile, name, nparam, params) \
```

Comment:
Actually, I have not started anything on this yet, got sidetracked with work on NormalizInterface. I only wanted to whip up to show what I had in mind, but I'll just scratch that for now, as we want to finish other things first.

I'll just say that a nice extra touch might be to add some extra safety for the arguments, by using additional helper macros similar to these:
```
#define ARGS0()             0, ""
#define ARGS1(a1)           1, #a1
#define ARGS2(a1,a2)        2, #a1 ", " #a2
#define ARGS3(a1,a2,a3)     3, #a1 ", " #a2 ", " #a3
#define ARGS4(a1,a2,a3,a4)  4, #a1 ", " #a2 ", " #a3 ", " #a4
```
and then write something like (made up):
```
   ...
   GVAR_FUNC("myfile.c", MyFunc, ARGS2(opL, opR)),
   ...
```